src/usr/share/gpgmailer/gpgkeyring.py

```diff
@@ -95,40 +94,34 @@ def is_trusted(self, fingerprint):
             trusted = True
 
         else:
-            self.logger.warn('Key %s is not trusted' % fingerprint)
+            self.logger.warn('Key %s is not trusted.' % fingerprint)
 
         return trusted
 
 
-    # Looks up a key fingerprint and returns the expiration date if it exists,
-    #   throws an exception if it is invalid or not in the keyring. If a format
-    #   is specified, the date will be expressed in that format.
+    # Returns a GPG key expiration date based on a key fingerprint. If the key fingerprint is
+    #   invalid or does not exist in the keyring, an exception is thrown.
     #
-    # fingerprint: A PGP key fingerprint
-    # date_format: A format string compatible with Python's strftime.
-    def get_key_expiration_date(self, fingerprint, date_format=None):
-        result = None
+    # fingerprint: A GPG key fingerprint.
+    # Returns the key's expiration date or None if no expiration date exist.
+    def get_key_expiration_date(self, fingerprint):
         self._fingerprint_is_valid(fingerprint)
 
-        result = self.fingerprint_to_key_dict[fingerprint]['expires']
-
-        if date_format:
-            result = datetime.datetime.fromtimestamp(result).strftime(date_format)
+        return self.fingerprint_to_key_dict[fingerprint]['expires']
```

Comment:
The previous code returns a string, but you're right, presentation isn't gpgkeyring's concern.